source/stylesheets/card-back.scss

```diff
@@ -0,0 +1,21 @@
+.card--back{
+  z-index: $base-z-index;
+  color: $white;
+
+  .card--back-content {
```

Comment:
The `--` naming convention suggests that this is a modifier of `card`, but it's probably an element that belongs to the `card` component.

Would `card__back-content` be more appropriate?

Also, are we gaining anything by nesting this?